docs/DeveloperGuide.md

```diff
@@ -624,3 +624,20 @@ testers are expected to do more *exploratory* testing.
       3. Launch BA€.
       
    3. **Expected:** A new addressbook.json file will be created and it will be empty.
+
+--------------------------------------------------------------------------------------------------------------------
+
+## **Appendix: Planned Enhancements**
+
+Team size: 5
+
+1. **Accept more symbols in names**: When adding or editing the details of a contact, the `NAME` field currently does accept certain symbols, such as `/` and `'`, there are other symbols that a user could desire to add to the `NAME` of a contact that are currently not accepted. This can include, but is not limited to, `,`, `-`, `(`, `)`, `.` or accented characters. We plan to adjust the validation RegEx of the `NAME` class to allow these other characters.
+2. **Improve duplicate contact detection**: When adding or editing the details of a contact, the duplicate detection feature currently compares `NAME` only. This could result in an inconvenience to the user should they choose to add multiple contacts with the same `NAME`. Hence, we plan to extend the current system of detecting duplicates based on `NAME` to one that detects duplicates by comparing multiple fields, such as but not limited to, `NAME`and `EMAIL`, and only considering a contact to be a duplicate of an existing one if both (or multiple) fields match in value.
+3. **Accept multiple phone numbers**: Currently, each contact can only have one `PHONE_NUMBER`, which may inconvenience some users who may wish to add contacts with multiple `PHONE_NUMBER`s. We plan to adjust the manner in which  `PHONE_NUMBER`s are stored, using a set similar to the manner in which `Tag` is stored, so that users are able to add more than one `PHONE_NUMBER`.
+4. **Accept more types of characters in phone numbers**: Currently, the `PHONE_NUMBER` field in each contact does not currently allow characters other than `+` and numbers. While as per the ITU-T, additional characters are not a strict necessity for for a `PHONE_NUMBER` to be usable globally, users in certain regions may prefer the inclusion of additional characters to improve readability. Additional characters can include, but are not limited to, ` ` (whitespace), `-`, `(` or `)`. We plan to add support for these additional characters by adjusting the validation RegEX in the Phone class.
```

Comment:
Should there be 2 "for"s?

"...additional characters are not a strict necessity for for a `PHONE_NUMBER`..."